src/App.vue

```diff
@@ -449,7 +449,7 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
       case SummaryActions.RESUME:
         // Save filing and return to dashboard
         await this.onClickSave()
-        this.goToDashboard(true)
+        // this.goToDashboard(true)
```

Comment:
Remove this before final commit.

Reply:
Absolutely, will do :) 
Useful for testing the filing structure and resuming drafts 😆

Reply:
fixed!